docs/shared/workspace/library-types.md

```diff
@@ -0,0 +1,103 @@
+# Library Types
+
+There are many different types of libraries in a workspace. In order to maintain a certain sense of order, we recommend having a small number of types, such as the below four (4) types of libraries:
```

Comment:
Asset libs?

Reply:
I added a note giving explicit permission for people to come up with their own types.